projects/packages/my-jetpack/_inc/utils/format-percentage.ts

```diff
@@ -0,0 +1,19 @@
+import { numberFormat } from '@automattic/jetpack-components';
+import type { FormatNumberFunction } from './types';
+
+const defaultConfig: Intl.NumberFormatOptions = {
+	style: 'percent',
+};
+
+const formatPercentage: FormatNumberFunction = ( number, config = defaultConfig ) => {
```

Comment:
Another nitpick - we could add JSDoc documentation for the formatPercentage function.